Pycloud Deployer/config.py

```diff
@@ -0,0 +1,30 @@
+import os
```

Comment:
The import statements are clear, but it's a common practice to group them in the following order: standard library imports, related third-party imports, and local application/library specific imports. Each group should be separated by a blank line.